dlls/cstrike/cstrike/CstrikeNatives.cpp

```diff
@@ -1707,6 +1708,49 @@ static cell AMX_NATIVE_CALL cs_set_c4_defusing(AMX* amx, cell* params)
 	return 1;
 }
 
+extern CreateNamedEntityFunc CS_CreateNamedEntity;
+extern UTIL_FindEntityByStringFunc CS_UTIL_FindEntityByString;
+
+// cs_create_entity(const classname[])
+static cell AMX_NATIVE_CALL cs_create_entity(AMX* amx, cell* params)
+{
+	if (CS_CreateNamedEntity > 0)
```

Comment:
I think this has to throw an error instead of silently failing, or at the very least leave some trace of this happening in the error/amxx logs. We should indicate that AMXX is at fault and not the plugin. Same goes for cs_find_ent_by_class().